cmd/gapit/sxs_video.go

```diff
@@ -107,7 +107,16 @@ func (verb *videoVerb) sxsVideoSource(
 	// be anything and the replayed frame will show the undefined framebuffer
 	// pattern.
 	// Permit the first run of frames to have no content.
-	permitNoMatch := true
+	permitNoMatch := false
+
+	for _, e := range events {
+		switch e.Kind {
+		case service.EventKind_Clear:
+			permitNoMatch = true
+		case service.EventKind_DrawCall:
+			permitNoMatch = true
```

Comment:
To avoid looping over all events, could this be refactored as the following?
```
for _, e := range events {
    if (e.Kind == service.EventKind_Clear) || (e.Kind == service.EventKind_DrawCall) {
        permitNoMatch = true
        break
    }
}
```

Reply:
Sure, nEvents should be reasonably small, but yes, that would be simpler.